addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -670,6 +670,24 @@ blockquote {
                     color: $dropdown-link-disabled-color !important;
                 }
             }
+            .btn-link {
+                color: $link-color;
+                &:hover {
```

Comment:
- Add a blank line between a rule properties and its sub-rules (I know I am annoying ^^)
```scss
.hello {
    a: b;

    .world {
        e: f;
    }
    .foo {
        c: d;
    }
}
```

- Instead of `$link-color`, should not this be `$btn-link-color`? (probably are the same but probably technically more correct).
- Add comments which mention the usecases where we need this (even if they are website ones and we are in a web_editor file) (for btn-link, text muted and the link instead text-muted)
- Note that the case with btn-link is in fact invalid DOM: a `<button>` inside a `<a>` (but the problem would be the same if the DOM structure was right, we can fix that one day in master).

Reply:
@qsm-odoo The only `$btn-link*` I could find is `$btn-link-disabled-color` which is indeed more accurate (the `o-btn-link-variant` mixin seems to only be used for variations of `btn-link`)

Reply:
Oh sorry, I was looking in the wrong bootstrap (the doc one)